### ISSUE TITLE
Backup/Restore threding

### ISSUE THREAD

**Issue by idofperion** [elastic/elasticsearch#17129]
Hi,

Currently snapshot, restore and snapshot delete are limited to one thread only.

This means I cannot restore while a snapshot /snapshot delete is in progress.

In large environments this may be a few hours a day.

My suggestions is to create a separate thread for restores that will not be blocked by running snapshot or deletes

10x

**Comment by clintongormley** [comment]
Duplicate of https://github.com/elastic/elasticsearch/issues/11148
